pkg/kn/commands/broker/create.go

```diff
@@ -31,7 +31,7 @@ var createExample = `
   kn broker create mybroker
 
   # Create a broker 'mybroker' in the 'myproject' namespace and with a broker class of 'Kafka'
-  kn broker create mybroker --namespace myproject --broker Kafka
```

Comment:
Run `./hack/build.sh` locally to generate the [docs](https://github.com/knative/client/blob/main/docs/cmd/kn_broker_create.md) and commit the generated changes in this PR, the build tests are complaining absence of (auto) generated docs

Reply:
Done.